src/ssu.cpp

```diff
@@ -25,6 +26,9 @@ Ssu::Ssu(QObject *parent) :
         QDBusConnection::systemBus(),
         parent )
 {
+    QDBusReply<QString> version = call(QStringLiteral("release"), false);
+    m_device_version = version;
+    qDebug() << "Device version:" << m_device_version;
```

Comment:
Users can override repo version. We should get real URL and use that, as described above